docs/other_libs.md

```diff
@@ -94,7 +94,7 @@ images            |  [jpeg-compressor](https://github.com/richgel999/jpeg-compre
 images            |  [easyexif](https://github.com/mayanklahiri/easyexif)                 | MIT                  | C++ |  2  | EXIF metadata extractor for JPEG images
 images            |**[cro_mipmap.h](https://github.com/thebeast33/cro_lib)**              | **public domain**    |C/C++|**1**| average, min, max mipmap generators
 math              |  [mm_vec.h](https://github.com/vurtun/mmx)                            | BSD                  |C/C++|**1**| SIMD vector math
-math              |**[ccVector.h](https://github.com/jobtalle/ccVector)                   | BSD                  |C/C++|**1**| Vector, quaternion and matrix math
+math              |**[ccVector.h](https://github.com/jobtalle/ccVector)**                 | BSD                  |C/C++|**1**| Vector, quaternion and matrix math
```

Comment:
If 3-clause BSD is an obstacle, I'll remove the license and make it public domain. Thank you for pointing this out.